app/src/main/res/layout/activity_media.xml

```diff
@@ -97,6 +97,7 @@
             android:layout_height="56dp"
             android:layout_toLeftOf="@+id/more"
             android:theme="@style/Ripple.List"
+            android:onClick="saveMedia"
```

Comment:
The save button already had an OnClickListener, and you also didn't account for the bottomsheet save ability.